src/server/rpc/procedures/baidu-maps/baidu-maps.js

```diff
@@ -0,0 +1,333 @@
+/**
+ * The BaiduMaps Service provides access to the Baidu Maps API to provide an alternative to Google Maps for Chinese users.
+ * For more information, check out http://lbsyun.baidu.com/index.php?title=static
+ *
+ * Terms of use: http://lbsyun.baidu.com/index.php?title=open/question
+ * @service
+ */
+'use strict';
+
+const logger = require('../utils/logger')('baidu-staticmaps');
+const request = require('request');
+const SphericalMercator = require('sphericalmercator');
+const geolib = require('geolib');
+const merc = new SphericalMercator({size:256});
+const CacheManager = require('cache-manager');
+const Storage = require('../../storage');
+
+// TODO: Change this cache to mongo or something (file?)
```

Comment:
this library supports other stores like filesystem store `cache-manager-fs` as is used in `ApiConsumer` so switching should be simple if a persistent storage is beneficial.